hypertrace-core-graphql-log-event-schema/src/main/java/org/hypertrace/core/graphql/log/event/dao/MockLogEventDao.java

```diff
@@ -0,0 +1,73 @@
+package org.hypertrace.core.graphql.log.event.dao;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.reactivex.rxjava3.core.Single;
+import java.util.List;
+import java.util.Map;
+import lombok.experimental.Accessors;
+import org.hypertrace.core.graphql.log.event.request.LogEventRequest;
+import org.hypertrace.core.graphql.log.event.schema.LogEvent;
+import org.hypertrace.core.graphql.log.event.schema.LogEventResultSet;
+
+public class MockLogEventDao implements LogEventDao {
+
+  @Override
+  public Single<LogEventResultSet> getLogEvents(LogEventRequest request) {
+    ObjectMapper objectMapper = new ObjectMapper();
+    Map<String, String> attributes = Map.of();
```

Comment:
Can you add some dummy data with few keys like `info`, `message`?

Reply:
Added